contrib/confluence/src/python/pants/contrib/confluence/util/confluence_util.py

```diff
@@ -140,11 +140,11 @@ def create_html_page(self, space, title, html, parent_page=None, **pageoptions):
     if self._content_format == 'markdown':
       content = '{html}\n\n%s\n\n{html}' % html
     elif self._content_format == 'xhtml':
-      content = '''<ac:macro ac:name="html">
-          <ac:plain-text-body><![CDATA[%s]]></ac:plain-text-body>
-          </ac:macro>''' % html
+      content = f'''<ac:macro ac:name="html">
+          <ac:plain-text-body><![CDATA[{html}]]></ac:plain-text-body>
+          </ac:macro>'''
     else:
-      raise ConfluenceError("Don't know how to convert %s to HTML" % format)
```

Comment:
format is undefined. not sure why a linter didn't alert on this.
probably because (format)(https://docs.python.org/3/library/functions.html#format) is a python builtin...